addon/background.js

```diff
@@ -49,3 +49,42 @@ browser.tabs.onUpdated.addListener((tabId, changeInfo, tabInfo) => {
             });
     }
 });
+
+async function freeze_active_tab(tab) {
```

Comment:
I don't feel like this should be named freeze_active_tab, since it freeze the tab passed in, whether it's active or not.

Reply:
That is a good point!